src/invidious/views/template.ecr

```diff
@@ -113,47 +113,9 @@
 
             <%= content %>
 
-            <footer>
-                <div class="pure-g">
-                    <div class="pure-u-1 pure-u-md-1-3">
-                        <span>
-                            <i class="icon ion-logo-github"></i>
-                        <% if CONFIG.modified_source_code_url %>
-                            <a href="https://github.com/iv-org/invidious"><%= translate(locale, "footer_original_source_code") %></a>&nbsp;/
-                            <a href="<%= CONFIG.modified_source_code_url %>"><%= translate(locale, "footer_modfied_source_code") %></a>
-                        <% else %>
-                            <a href="https://github.com/iv-org/invidious"><%= translate(locale, "footer_source_code") %></a>
-                        <% end %>
-                        </span>
-                        <span>
-                            <i class="icon ion-ios-paper"></i>
-                            <a href="https://github.com/iv-org/documentation"><%= translate(locale, "footer_documentation") %></a>
-                        </span>
-                    </div>
-
-                    <div class="pure-u-1 pure-u-md-1-3">
-                        <span>
-                            <a href="https://github.com/iv-org/invidious/blob/master/LICENSE"><%= translate(locale, "Released under the AGPLv3 on Github.") %></a>
-                        </span>
-                        <span>
-                            <i class="icon ion-logo-javascript"></i>
-                            <a rel="jslicense" href="/licenses"><%= translate(locale, "View JavaScript license information.") %></a>
-                        </span>
-                        <span>
-                            <i class="icon ion-ios-paper"></i>
-                            <a href="/privacy"><%= translate(locale, "View privacy policy.") %></a>
-                        </span>
-                    </div>
-
-                    <div class="pure-u-1 pure-u-md-1-3">
-                        <span>
-                            <i class="icon ion-ios-wallet"></i>
-                            <a href="https://invidious.io/donate/"><%= translate(locale, "footer_donate_page") %></a>
-                        </span>
-                        <span><%= translate(locale, "Current version: ") %> <%= CURRENT_VERSION %>-<%= CURRENT_COMMIT %> @ <%= CURRENT_BRANCH %></span>
-                    </div>
-                </div>
-            </footer>
+            <% if buffer_footer %>
+                <div id="footer_buffer"></div>
+            <% end %>
```

Comment:
It's a bit hard to explain but its mostly to give the contents some breathing room on select pages. It looks too cramped and shows too much information right-away imo. A block footer like so is intended to be scrolled down as to not overshadow the main contents of the page. See the examples below: 

Without buffer (visible without scrolling): 
![without buffer](https://github.com/iv-org/invidious/assets/70992037/2aea14d5-246e-4719-9b40-02d75e679655)


With buffer (requires scrolling):
![with buffer](https://github.com/iv-org/invidious/assets/70992037/f106c211-6882-4fc8-bb7d-297ca9747122)


You can find some examples on sites like stackoverflow.com, vox.com, etc.

Reply:
Ah, I see! I think that 50vh may be a bit too large. what about 1/4th of that?
Also, can you put the footer div only on the selected .ecr views, rather than in the template?